hibernate-reactive-core/src/test/java/org/hibernate/reactive/ReactiveStatelessProxyUpdateTest.java

```diff
@@ -83,13 +77,16 @@ public void testUnfetchedEntityException(TestContext context) {
 							return s.update( entityFromDb );
 						} )
 				)
+				.onFailure().recoverWithUni( throwable -> {
+					ReactiveAssertions.assertException(
```

Comment:
You can use assert thrown or this test won't fail in case we don't throw an exception